Tests fail consisnently when using `golang-migrate/migate`

We have some other PRs up for review that attempt to add integration tests for the database. One of the tools these tests use is `golang-migrate/migrate`.

The go-specific CI runs are failing with errors similar to:

```
 Status: Downloaded newer image for postgres:latest
bb8e9bb204c8722103e672ff7f8523392f3437f8dc40c07c2dff1de73a140d08
go: downloading github.com/golang-migrate/migrate/v4 v4.15.2
go: downloading github.com/golang-migrate/migrate v3.5.4+incompatible
go: downloading github.com/hashicorp/go-multierror v1.1.1
go: downloading go.uber.org/atomic v1.7.0
go: downloading github.com/hashicorp/errwrap v1.1.0
# github.com/rhmdnd/compserv/tests/integration
FAIL	github.com/rhmdnd/compserv/tests/integration [setup failed]
Error: tests/integration/integration_test.go:8:2: cannot find package "." in:
	/home/runner/work/compserv/compserv/vendor/github.com/golang-migrate/migrate/v4
```

It seems clear the issue stems from a dependency problem with migrate.

--- a/.gitignore
+++ b/.gitignore
@@ -9,8 +9,5 @@ configs/config.yaml
 terraform/.terraform*
 terraform/terraform.*
 
-# Ignore the golang-migrate/migrate binary
-migrate
-
 *.swp
 .idea

